src/script/app.js

```diff
@@ -0,0 +1,10 @@
+let scroll1 = window.pageYOffset;
+window.onscroll = function () {
+    let scroll2 = window.pageYOffset;
+    if (scroll1 > scroll2) {
+        document.querySelector('nav').style.top = "0";
+    } else {
+        document.querySelector('nav').style.top = "-80px";
+    }
+    scroll1 = scroll2;
+}
```

Comment:
It looks like it's all copied from the internet. Read the [addEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) documentation, which is currently the preferred way of setting up the event handlers via JavaScript.